packages/repository-tests/src/crud/relations/fixtures/models/address.model.ts

```diff
@@ -1,33 +1,45 @@
 // Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/repository
+// Node module: @loopback/repository-tests
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {belongsTo, Entity, model, property} from '../../..';
+import {belongsTo, Entity, model, property} from '@loopback/repository';
 import {Customer, CustomerWithRelations} from './customer.model';
 
-@model()
+@model({
+  settings: {
+    strictObjectIDCoercion: true,
+  },
+})
 export class Address extends Entity {
+  @property({
+    type: 'string',
+    id: true,
+    generated: true,
+  })
+  id: string;
   @property({
     type: 'string',
   })
   street: string;
   @property({
     type: 'string',
-    id: true,
+    default: '12345',
```

Comment:
If we don't set the default value, MySQL somehow would have `undefined` instead of `null`

Reply:
> The CI tests are failing and seem related

They were just timeout failures. They're gone now.

Reply:
> MySQL somehow would have undefined instead of null

This behavior sounds reasonable to me, 🤔 , why is it a problem? Some tests expect `null` instead of `undefined`? Or any API calls will be broken?

Reply:
If nothing is provided then loopback uses `undefined` but mysql uses `null`. And if we use `null` in lb4, it's not the same thing as it's in MySQL. That's why we use these default values.

Reply:
It's a well-know problem (feature?) that NoSQL databases like `memory` and MongoDB represent missing values are `undefined`, while SQL databases use `NULL`. We have `features.emptyValue` to use in tests to deal with that design.

Please remove `default` fields from the property definitions and modify the checks to use `feature.emptyValue` instead of the default or `undefined`. 

See e.g. this test for inspiration:

https://github.com/strongloop/loopback-next/blob/6ad0bb59c71ea2356cb951da786bdbff246b47e7/packages/repository-tests/src/crud/replace-by-id.suite.ts#L73-L81